microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Build PWAs for the sidebar in Microsoft Edge
+description: How to build your PWA for installation in the Microsoft Edge sidebar.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+ms.date: 04/27/2023
+---
+# Build PWAs for the sidebar in Microsoft Edge
+
+![Experimental feature](../../media/experimental-tag.png)
+
+Progressive Web Apps (PWAs) can opt-in to be pinned to the sidebar in Microsoft Edge.
+
+The sidebar in Microsoft Edge allows users to easily access popular websites and utilities alongside their browser tabs. The content in the sidebar augments the user's primary task by enabling side-by-side browsing and minimizing the need to switch contexts between browser tabs.
+
+By signaling intent to be pinned to the sidebar, your PWA gets the following benefits:
+
+* Your PWA can be promoted in the Microsoft Edge sidebar store.
+* Your PWA can detect whether it is running in the Microsoft Edge sidebar or not.
+* Your PWA defines its own style and layout to provide a user-friendly sidebar experience.
+
+
+<!-- ====================================================================== -->
+## Enable sidebar support of PWAs in Microsoft Edge
+
+Experimental support for PWAs that can target the sidebar in Microsoft Edge starts with Microsoft Edge 112. Some parts of the feature are available starting with Microsoft Edge 114, such as the ability to detect if your PWA is running in the sidebar by using User Agent Client Hints.
+
+To locally enable PWA sidebar support in Microsoft Edge:
+
+1. Download and install the Canary channel of Microsoft Edge from the [Microsoft Edge Insider website](https://www.microsoftedgeinsider.com/download).
+
+1. Depending on your operating system, do either of the following to start Microsoft Edge with sidebar support for PWAs:
+
+    * On Windows:
+        
+        * Find the path where Microsoft Edge Canary is installed on your device. For example: `C:\Users\your_name\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe`.
+        * Enter **cmd** in the Windows search text box and select the **Command Prompt** app.
+        * Enter `"C:\Users\your_name\AppData\Local\Microsoft\Edge SxS\Application\msedge.exe" --enable-features=msWebAppManifestSidePanel,msAddEdgeSidePaneBrandUA`.
+
+    * On macOS:
+
+        * Find the path where Microsoft Edge Canary is installed on your device. For example: `/Applications/Microsoft Edge Canary.app`.
+        * Open the **Terminal** app.
+        * Enter `/Applications/Microsoft Edge Canary.app --enable-features=msWebAppManifestSidePanel,msAddEdgeSidePaneBrandUA`.
```

Comment:
```suggestion
        1. Find the path where Microsoft Edge Canary is installed on your device. For example: `/Applications/Microsoft Edge Canary.app`.
        1. Open the **Terminal** app.
        1. Enter `/Applications/Microsoft Edge Canary.app --enable-features=msWebAppManifestSidePanel,msAddEdgeSidePaneBrandUA`.
```

Reply:
result according to VS Code's preview with Learn pack:
![image](https://user-images.githubusercontent.com/45407486/234967217-554b8f99-643c-4c49-8648-cbe5879b5dcf.png)